app/models/post.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
+  default_scope -> { order(created_at: :desc) }
+  validates :user_id, presence: true
```

Comment:
- [ ] `belongs_to :user` adds presence validation to `user_id` by default.
[check this](https://blog.bigbinary.com/2016/02/15/rails-5-makes-belong-to-association-required-by-default.html)